src/components/ScoreBox/WordScoreList.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import TotalScore from './TotalScore'
+const WordScoreList = (props) => {
+  const {words, scores, totalScore}  = props
+
+  const wordsList = words.map(function(word, index) {
+    return (<li key={word.toString()+index}>{word}</li>)
```

Comment:
in React, you need to specify unique key for each component if your component is part of an array of the children props